### ISSUE TITLE
No Longer Current

### ISSUE THREAD

**Issue by Goahnary** [jackyliang/Pollen-Buddy-PHP#8]
Jacky,

First off thanks for making this. You are a life saver! Currently I have just implemented this on a test page. Unfortunately it seems to be stuck in December 2016. Is this an issue with Wunderground? Maybe I haven't set something correctly? Here is my code currently for the test page:

```php
<?php
    require_once("Pollen-Buddy-PHP/PollenBuddy.php");
?>
<!DOCTYPE html>
<html>
<head>
	<title>testing page</title>
</head>
<body>
<?php
	$data = new PollenBuddy(42101);
	// echo $data->getSiteHTML();
	var_dump($data->getFourDayForecast());
?>
</body>
</html>
```

This currently returns:

`array(4) { ["December 14, 2016"]=> string(3) ".10" ["December 15, 2016"]=> string(3) ".10" ["December 16, 2016"]=> string(3) ".10" ["December 17, 2016"]=> string(3) ".10" }`

Please let me know. Thanks, again!

**Comment by jackyliang** [comment]
Hey - I actually am not sure. This library has not been maintained in awhile. If you want to check the source (should be pretty straightforward) and see what's going on, totally go ahead and do that. I have a feeling it might just be Wunderground being the issue.

EDIT:

Yeah, looks like the data for your specific zipcode is stuck in December 2016 for some reason. Maybe you could reach out to Wunderground? https://www.wunderground.com/DisplayPollen.asp?Zipcode=42101

I just checked my zipcode and that too also was stuck in December

**Comment by Goahnary** [comment]
Thanks! I will check the source and see what I can find! Looks pretty simple.

⁣Sent from BlueMail ​

**Comment by Goahnary** [comment]
Yes, wunderground seems to be the problem. I will see what I can do to remedy the issue.

EDIT:
It looks like every location is stuck in December 2016 at the moment.

**Comment by Goahnary** [comment]
I emailed wunderground and this is what they had to say:

>Hello,
>
>Thank you for letting us know. That's an older page and it's going to the wrong place. We'll fix it!
>
>In the meantime, if you go to wunderground.com and search for any location, taking you to its forecast page, click "Health" tab just above the current conditions and you'll get pollen info.
>
>Thank you,
>William
>Wunderground Support

The new health tab seems to have a consistent logic so getting the data shouldn't be too difficult. I am going to start working on this soon!

**Comment by jackyliang** [comment]
Thanks! Feel free to submit a PR and I'll merge it if it looks good.

**Comment by Goahnary** [comment]
Will do!

⁣Sent from BlueMail ​
